.github/workflows/backend-ci.yml

```
@@ -0,0 +1,94 @@
+name: Backend CI
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'backend/**'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'backend/**'
+
```

一応workflow_dispatchも追加しておいた方が便利かなと思いました

```suggestion
  workflow_dispatch:

```